installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
To answer Atchut's question a little further, yes, these limits can be placed at the group level.
To answer Narine's question, no. The owners of the the nxautomation user should place limits if desired, since they have any and all context for what the limits should be. For now, the customer is only looking for limits on the omsagent user.